mono/metadata/cominterop.c

```diff
@@ -2335,18 +2335,18 @@ cominterop_get_managed_wrapper_adjusted (MonoMethod *method)
 
 #ifndef DISABLE_JIT
 	if (!preserve_sig)
+	{
```

Comment:
I like this brace placement, but around here it is just me.

Reply:
Fixed.